Sources/ProjectDescription/Product.swift

```diff
@@ -21,5 +21,5 @@ public enum Product: String, Codable {
 //    case tvExtension
 //    case messagesApplication
 //    case messagesExtension
-//    case stickerPack
+    case stickerPack = "sticker_pack"
```

Comment:
perhaps here `stickersPackExtension` ? (to help distinguish it from `stickersPackApp`)

Reply:
Agree here

Reply:
Sure, by the way there is no stickersPackApp currently in XcodeProj

Reply:
😬 we'll have to add it at some point. Feel free to open a PR there :)